react/store/ReturnRequest/StoreReturnDetails.tsx

```diff
@@ -40,7 +42,14 @@ const StoreReturnDetails = () => {
             to: '#/my-returns',
           })
         }}
-      />
+      >
+        <AlertProvider>
```

Comment:
We don't need to do it right now. Just something for us to think about regarding the structure of the app. Since we are using it now here, we might want to bring it outside the admin folder.